export.js

```diff
@@ -37,25 +37,51 @@ if(platform == "linux"){
     options.executablePath = "/Applications/Google\ Chrome.app/Contents/MacOS/Google\ Chrome"
 }
 
-
 async function main() {
     try{
         if(platform == "linux"){
             xvfb.startSync()
         }
-        var url = process.argv[2],
-            exportname = process.argv[3], 
-            duration = process.argv[4],
-            convert = process.argv[5]
-
-        if(!url){ url = 'https://www.mynaparrot.com/' }
-        if(!exportname){ exportname = 'export.webm' }
-        if(!duration){ duration = 10 }
-        if(!convert){ convert = false }
         
+        var url = process.argv[2];
+        if(!url){
+            console.warn('URL undefined!');
+            process.exit(1);
+        }
+        // Verify if recording URL has the correct format
+        var urlRegex = new RegExp('^https?:\\/\\/.*\\/playback\\/presentation\\/2\\.0\\/playback\\.html\\?meetingId=[a-z0-9]{40}-[0-9]{13}');
```

Comment:
Please change validation for playback.html because some user may use custom file. Some of my client has their own designed page.

Reply:
I think a possible solution would be to add a variable in config.js file ("playbackFlie", for example) that indicates the name of the file and modify the regular expression to use that variable:

`var urlRegex = new RegExp('^https?:\\/\\/.*\\/playback\\/presentation\\/2\\.0\\/' + config.playbackFile + '\\?meetingId=[a-z0-9]{40}-[0-9]{13}');`